src/DropDownMenu/DropDownMenu.js

```diff
@@ -114,6 +114,11 @@ class DropDownMenu extends Component {
      */
     menuStyle: PropTypes.object,
     /**
+     * If true, `value` must be an array and the menu will support
+     * multiple selections.
+     */
+    multiple: PropTypes.bool,
+    /**
```

Comment:
I can't think of a use case for multiple items in a "regular" drop down menu, but I might be wrong. If that's the case `@ignore` should works.